tests/ssl-opt.sh

```diff
@@ -1401,6 +1401,22 @@ run_test    "Renegotiation: client-initiated" \
             -s "=> renegotiate" \
             -S "write hello request"
 
+# Checks if the server parses the SigAlgs Extension. Valid only if an MD
+# algorithm stronger than SHA1 is enabled in config.h
+run_test    "Renegotiation: Signature Algorithms regression, client" \
```

Comment:
"regression" is a name we will forget the meaning of easily. Please consider a better name.

Reply:
Might also be useful to say "client initiated" instead of just "client" (and likewise for server).